dluhc-component-library/src/components/maps/MapComponent.tsx

```diff
@@ -0,0 +1,73 @@
+import MapContainer from "./mapContainer";
+import BaseMap from "./baseMap";
+import DrawingLayer from "./drawingLayer";
+import { CSSProperties, useMemo } from "preact/compat";
+
+interface MapComponentProps {
+  baseMapProps?: BaseMapProps;
+  drawingMapProps?: DrawingMapProps;
+  id?: string;
+  className?: string;
+  style?: CSSProperties;
+}
+
+interface BaseMapProps {
+  lat?: number;
+  lng?: number;
+  zoom?: number;
+  drawingMode?: boolean;
```

Comment:
booleans should start with `is` so that its obvious that its a boolean, this reads like it should be an enum without it.